envs/py311_uscuni_simplification.yml

```diff
@@ -1,9 +1,9 @@
-name: py312_uscuni_simplification
+name: py311_uscuni_simplification
```

Comment:
Can we shorten the name? It is a pain to type it to activate the env :D

Reply:
sure, lazy bones. any suggestions?

Reply:
FWIW, I create aliased functions in my shell that calls these up automatically without having to type them in.

Reply:
re: naming – explicit >> implicit

Reply:
I had my called `simplification` but I can also easily override the name...

Reply:
That works for me.

Reply:
Shall I update the name of the env file and the env name itself?

Reply:
I suppose that if we stick to a single env only, it can be an `environment.yml` in root.

Reply:
fair enough. I will update and re-push.